src/app/component/result/result.component.ts

```diff
@@ -37,13 +39,13 @@ export class ResultComponent implements OnInit, OnDestroy {
   resultData() {
     this.store
       .select((state: any) => state.quiz.latestQuizResult)
-      .pipe(distinctUntilChanged(),takeUntil(this.destroyer$))
+      .pipe(distinctUntilChanged(), takeUntil(this.destroyer$))
       .subscribe((data) => {
         this.userData = data;
       });
     if (!this.userData) {
       this.router.navigateByUrl('dashbaord');
-      this.toastService.showErrorMessage('No Quiz Played Yet')
+      this.toastService.toastMessage(RESULT_QUIZ,'fa-solid fa-triangle-exclamation');
```

Comment:
if you have already passed variable in constant then why your using like this  ?